src/frontend/komponenter/Felleskomponenter/HeaderMedSøk/OpprettFagsakModal.tsx

```diff
@@ -95,7 +97,19 @@ const OpprettFagsakModal: React.FC<IOpprettFagsakModal> = ({
     );
     const [visFeilmelding, settVisFeilmelding] = useState(false);
     const [valgtSamhandler, settValgtSamhandler] = useState<ISamhandlerInfo | undefined>(undefined);
-    const { onSubmitWrapper, samhandlerSkjema } = useSamhandlerSkjema();
+    const [spinner, settSpinner] = useState(false);
+    const { onSubmitWrapper, samhandlerSkjema } = useSamhandlerSkjema(
+        _r => {
```

Comment:
Hvis du ikke trenger verdien inni funksjonen tenker jeg det er bedre å ikke ta den inn
```suggestion
        () => {
```

Reply:
Samme på linje 107